mlflow/entities/logged_model.py

```diff
@@ -53,6 +53,23 @@ def __init__(
         self._metrics: Optional[list[Metric]] = metrics
         self._model_uri = f"models:/{self.model_id}"
 
+    def __repr__(self) -> str:
+        return "LoggedModel({})".format(
+            ", ".join(
+                f"{k}={v!r}"
+                for k, v in self
+                if (
+                    k
+                    not in [
+                        # These fields can be large and take up space on the notebook or terminal
+                        "tags",
+                        "params",
+                        "metrics",
```

Comment:
we could truncate and show the first N items, but users can just do `model.{params|metrics|tags}` if they want to view them.